### DISCUSSION TITLE
JK BMS cell voltage difference alarm

### DISCUSSION THREAD

**Discussion by squowse** [Louisvdw/dbus-serialbattery#1050] [Q&A] [answered]
Ive given up on the JBD bms for now with its erroneous “consumed Ah”.

I have managed to connect a JK-b2a24s15p by Bluetooth without too much bother.

Only issue is frequent “cell imbalance” alarms. Is there a way to deactivate these? Maybe using the JK app on Bluetooth? I’ve never noticed it on the app.

Easy way to release the Bluetooth connection from the Cerbo temporarily. The app is not showing the BMS. Because its already connected to the Cerbo i suspect.

**Comment by mr-manuel** [comment, marked as answer]
It's definded in the driver. You can change it manually:

https://github.com/Louisvdw/dbus-serialbattery/blob/59a6d00c096c3e708aaf9cd827275c6500450374/etc/dbus-serialbattery/bms/jkbms_ble.py#L227-L237

**Reply by squowse** [reply]
Ah thanks. I will investigate
